WordPress/src/main/java/org/wordpress/android/ui/jetpack/JetpackCapabilitiesUseCase.kt

```diff
@@ -0,0 +1,90 @@
+package org.wordpress.android.ui.jetpack
```

Comment:
I renamed the file - I'm not sure why GitHub is treating it as remove and add 🤷 .

Reply:
I usually try to rename single file and commit at a time. I see `JetpackCapabilitiesUseCaseTest` is renamed as well. I'm not sure if this's the issue here though.

Reply:
Hmm I used AS wizard for the rename - and the wizard renamed even the related tests. I've used the same approach in the past and it worked. I'm not sure what happened here 🤷 .

Reply:
🤷‍♀️